code/.hlint.yaml

```diff
@@ -54,6 +54,7 @@
 # Ignore some builtin hints
 # - ignore: {name: Use let}
 # - ignore: {name: Use const, within: SpecialModule} # Only within certain modules
+  - ignore: {name: "Use camelCase"}
```

Comment:
I believe you just need to remove the leading whitespace in front of the `-` so that it can be parsed for hlint.

Reply:
Ah okay I'll try that. Thanks!

Reply:
@Ant13731, you might want to run HLint before doing a pull request.  I think there is a make rule for hlint?

Reply:
make hlint didn't work but I think the program passed the build test for hlint at least.